dist/metricsgraphics.js

```diff
@@ -1986,13 +1986,18 @@ function markers(args) {
                 .attr('y2', function() {
                     return args.height - args.bottom - args.buffer;
                 })
+                .attr('class', function (d) {
+                    return d.lineclass;
+                })
                 .attr('stroke-dasharray', '3,1');
 
         gm.selectAll('.mg-markers')
             .data(args.markers.filter(inRange))
             .enter()
             .append('text')
-                .attr('class', 'mg-marker-text')
+                .attr('class', function (d) {
+                    return d.textclass ? 'mg-marker-text' : 'mg-marker-text ' + d.textclass;
```

Comment:
Shouldn't this be `return d.textclass ? 'mg-marker-text ' + d.textclass : 'mg-marker-text'`?
If textclass is not set, the current statement tries to use it, which I assume would add `undefined` to the class.